data.sql

```diff
@@ -63,3 +63,38 @@ update animals set owner_id = 4 where name = 'Blossom';
 update animals set owner_id = 5 where name = 'Angemon';
 update animals set owner_id = 5 where name = 'Boarmon';
 
+INSERT INTO vets(id,name,age,date_of_graduation)
+values(1,'William Tatcher',45,'2000-04-23');
+INSERT INTO vets(id,name,age,date_of_graduation)
+values(2,'Maisy Smith',26,'2019-06-17');
```

Comment:
- [ ] Please you are inserting the wrong date_of_graduation for Vet Maisy Smith who graduated Jan 17th, 2019 but you inserted the date as June 17th, 2019 so make sure you fix this.